build/bin/install-dependencies

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+usage () {
+cat <<HELP_USAGE
+$0 
+
+Install dependency packages required to run an exitnode.
+HELP_USAGE
+}
+
+set -e
+
+release_info="$(cat /etc/*-release)"
+echo "release_info=$release_info"
+release_name="$(echo "$release_info" | grep ^NAME= | cut -d'=' -f2)"
+echo "release_name=$release_name"
+
+
+# Upgrade kernel on OS that require it
+echo "release_name=$release_name"
+if [ "$release_name" == '"Ubuntu"' ]; then
+  echo "OS is Ubuntu. Need to upgrade kernel"
+  # This would NOT work on Debian 8.10, (@TODO test others).
```

Comment:
nice! By "would NOT work" you mean something like would not be executed correct?